src/utils/front-matter.js

```diff
@@ -5,38 +5,54 @@ const {
   builders: { hardline, concat, markAsRoot },
 } = require("../document");
 
+const yamlStart = "---";
+const yamlEnd = "...";
```

Comment:
Can you rename and add comment about `...` end  delimiter?

```suggestion
const YAML_DELIMITER = "---";
// In some markdown prosessor such as pandoc,
// "..." can be used as the end delimiter for front-matter.
const YAML_END_DELIMITER = "...";
```

Reply:
done!